go.html.markdown

```diff
@@ -133,6 +135,11 @@ can include line breaks.` // Same string type.
 	// Unused variables are an error in Go.
 	// The underscore lets you "use" a variable but discard its value.
 	_, _, _, _, _, _, _, _, _, _ = str, s2, g, f, u, pi, n, a3, s4, bs
+	// Usually you use it to ignore one of the return values of a function
+	file, _ := os.Create("output.txt") // Should never ever cause an error
```

Comment:
Good point. I'll rephase to "we expect this call to be successful, so we're ignoring the error"